### ISSUE TITLE
Support date-time conversion from unix epoch

### ISSUE THREAD

**Issue by balki** [mikefarah/yq#1535]
Many software write timestamp in unix epoch format to logs. E.g.

```json
{"level":"info","ts":1675020196.476687,"logger":"app.main","msg":"handled request","status":0}
```
Need a way to convert `1675020196.476687` to human readable value. If there is already a way, please add it to the [docs](https://mikefarah.gitbook.io/yq/operators/datetime). 

**Using jq**

```bash
❯ cat /tmp/ex.json | jq '.ts |= strflocaltime("%Y-%m-%dT%H:%M:%S %Z")' 
{
  "level": "info",
  "ts": "2023-01-29T14:23:16 EST",
  "logger": "app.main",
  "msg": "handled request",
  "status": 0
}
```

**Comment by mikefarah** [comment]
Good feature request, I'll add it when I get the time

**Comment by balki** [comment]
Great! Thanks!

**Comment by mikefarah** [comment]
Fixed in v4.31.1 https://mikefarah.gitbook.io/yq/operators/datetime#from-unix
